src/coredump/coredumpctl.c

```diff
@@ -467,6 +483,19 @@ static void analyze_coredump_file(
         *ret_size = UINT64_MAX;
 }
 
+static int get_absolute_filename(sd_journal *j, char **filename) {
+        if (!j->prefix)
+                return 0;
+
+        char *absolute_filename = NULL;
+
+        absolute_filename = path_join(j->prefix, *filename);
```

Comment:
why `j->prefix` instead of `arg_root`?

Reply:
I thought it would be better to get all the information from the journal structure, but it just makes the code more difficult to understand. I will change it.